web/src/inspector.js

```diff
@@ -222,13 +274,18 @@ export class Inspector extends EventTarget {
       if (Inspector._needsExtraInfo(torrents)) {
         fields.push(...Torrent.Fields.InfoExtra);
       }
+      if (this.settings_dirty) {
```

Comment:
Currently, torrents don't resync per-torrent settings (as they aren't displayed anywhere) so this mechanism provokes a one-time resync for these fields for all selected torrents.